README.md

```diff
@@ -102,13 +102,15 @@ lua <<EOF
     })
   })
 
-  -- Use buffer source for `/` (if you enabled `native_menu`, this won't work anymore).
-  cmp.setup.cmdline('/', {
-    mapping = cmp.mapping.preset.cmdline(),
-    sources = {
-      { name = 'buffer' }
-    }
-  })
+  -- Use buffer source for `/` and `?` (if you enabled `native_menu`, this won't work anymore).
+  for _,v in pairs({ '/', '?' })
```

Comment:
@amarakon I think this needs `do` at the end: `for _,v in pairs({ '/', '?' }) do`.

Reply:
Yes, it does. I forgot to put it, that is my bad.